src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -240,68 +240,65 @@ const ContentListTable = () => {
   } = data;
 
   const rowActions = useCallback(
-    (rowData: ContentItem): IAction[] =>
-      isRedHatRepository
-        ? features?.snapshots?.accessible
-          ? [
+    (rowData: ContentItem): IAction[] => {
+      const triggerSnapshotAction = {
+        isDisabled: actionTakingPlace || rowData?.status === 'Retrying',
+        title: 'Trigger Snapshot',
+      };
```

Comment:
You still haven't added the API call to this, you have simple added a button to the UI. 
No logic exists to do that at this time. 

As well, the majority of this code isn't necessary.
You can simply add your : 
`
{
        isDisabled: actionTakingPlace || rowData?.status === 'Retrying',
        title: 'Trigger Snapshot',
},

to the original array without moving all of this around for no reason.

 `